scripts/fast-reboot

```diff
@@ -32,6 +36,24 @@ INITRD=$(echo $KERNEL_IMAGE | sed 's/vmlinuz/initrd.img/g')
 
 sonic_asic_type=$(sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type)
 
+# Install new FW for mellanox platforms before control plane goes down
+# So on boot switch will not spend time to upgrade FW increasing the CP downtime
+if [[ "$sonic_asic_type" == "mellanox" ]]; then
+    echo "Prepare MLNX ASIC to fast reboot: install new FW if required"
+
+    MLNX_EXIT_SUCCESS="0"
+    MLNX_EXIT_ERROR="1"
```

Comment:
>1 [](start = 21, length = 1)

Exit code 1 is already used. Do you want another one?

Reply:
changed to 100 (as it is in master)